packages/core/upload/admin/src/utils/urlsToAssets.js

```diff
@@ -2,6 +2,14 @@ import axios from 'axios';
 import { AssetSource } from '../constants';
 import { typeFromMime } from './typeFromMime';
 
+/**
+ * The URL might have query parameters.
+ * Remove them to have a clean URL instead to use as a filename.
+ */
+function cleanUrl(url) {
+  return new URL(url).pathname.substring(1);
+}
```

Comment:
why not using the last element of the path?
```js
  return new URL(url).pathname.split('/').pop();
```
Example:
![Capture d’écran 2022-08-09 à 14 39 27](https://user-images.githubusercontent.com/11708220/183649094-b4b87f1e-88b4-4f36-9871-471ae10d7cde.png)

Reply:
Sounds good to me too.

Reply:
will change that, you are right 😃

Reply:
![image](https://user-images.githubusercontent.com/20578351/183698600-7a121c9e-9e98-40b6-aef2-708bcc8c0789.png)

Reply:
@Marc-Roig it's a bit nit-picking and I think we should merge this PR and apply it after the PR, but I think `cleanURL` doesn't make sense any longer as method name. It's more a `getFilenameFromURL` or something similar :) Wdyt?

Reply:
you are right! changed